.env-issuer.sample

```diff
@@ -0,0 +1,31 @@
+ISSUER_SERVER_URL=localhost:3001
+ISSUER_SERVER_PORT=3001
+ISSUER_NATIVE_PROOF_GENERATION_ENABLED=true
+ISSUER_PUBLISH_KEY_PATH=pbkey
+ISSUER_ONCHAIN_PUBLISH_STATE_FRECUENCY=1m
+ISSUER_ONCHAIN_CHECK_STATUS_FRECUENCY=1m
+ISSUER_DATABASE_URL=postgres://polygonid:polygonid@localhost:5432/platformid?sslmode=disable
+ISSUER_LOG_LEVEL=-4
+ISSUER_LOG_MODE=2
+ISSUER_HTTPBASICAUTH_USER=user
+ISSUER_HTTPBASICAUTH_PASSWORD=password
+ISSUER_KEY_STORE_ADDRESS=http://vault:8200
+ISSUER_KEY_STORE_PLUGIN_IDEN3_MOUNT_PATH=iden3
+ISSUER_REVERSE_HASH_SERVICE_URL=http://localhost:3001
+ISSUER_REVERSE_HASH_SERVICE_ENABLED=false
+ISSUER_ETHEREUM_URL=https://polygon-mumbai.g.alchemy.com/v2/XXXX
```

Comment:
Should be a placeholder like `<e.g. https://polygon-mumbai.g.alchemy.com/v2/XXXX>` since this is an invalid URL.